### ISSUE TITLE
Engine crashes after closing project

### ISSUE THREAD

**Issue by mazlum1414** [getnamo/socketio-client-ue4#300]
![Adsız](https://user-images.githubusercontent.com/83881717/150179663-ed771deb-ef98-4269-9ae8-9fae62d596d6.png)

**Comment by mazlum1414** [comment]
hello I'm trying to make a multiplayer project, if I put ChatWinActor on the map and start the game as 3player or more, and close it, the engine crashes immediately, it wasn't like this before

**Comment by mazlum1414** [comment]
![Adsız](https://user-images.githubusercontent.com/83881717/150180074-9d0d926b-73c4-4bf4-8768-13bdf631a5fc.png)

**Comment by mazlum1414** [comment]
![Adsız](https://user-images.githubusercontent.com/83881717/150180199-93a949d2-f414-4fca-bea6-968399816ba6.png)

**Comment by mazlum1414** [comment]
![Adsız](https://user-images.githubusercontent.com/83881717/150180373-967d3f13-0716-438e-ad0f-018964987756.png)

**Comment by mazlum1414** [comment]
![Adsız](https://user-images.githubusercontent.com/83881717/150180474-55a86371-c454-49fe-baf1-4a866cbfdaa0.png)

**Comment by mazlum1414** [comment]
sorry for the tag @getnamo

**Comment by getnamo** [comment]
Error specifies trying to generate object after garbage collection (which I assume happens due to project closing), this is caused by some SIOJson value being generated during shutdown procedure, which according to the stack happens because you get a response over the network. You may wish to try to disconnect earlier before project close and see if the problem still persists.

**Comment by mazlum1414** [comment]
> Error specifies trying to generate object after garbage collection (which I assume happens due to project closing), this is caused by some SIOJson value being generated during shutdown procedure, which according to the stack happens because you get a response over the network. You may wish to try to disconnect earlier before project close and see if the problem still persists.

yes, when I disconnect from the server and close the project, the crash disappears, what could be the reason, how can I solve it?
